src/ListView/ListView.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+/**
+ * ListView Component for Patternfly React
+ */
+const ListView = ({ children, listViewClass }) => (
+  <div className={listViewClass}>
```

Comment:
I think that a css class which defines list (list-pf) should not be overridable as it is basically the element which defines the component. Although it is deffinitely valid to allow user to add additional css classes to that. IMHO it is best to use standard className prop for this and combine it in component:

<div className={[ 'list-pf', className ].filter(i => i).join(' ')}>